CHANGELOG.unreleased.md

```diff
@@ -15,6 +15,7 @@
 - [Proxies] Fix `xapi.getOrWaitObject is not a function` is not a function during deployment
 - [REST API] Fix empty object's tasks list
 - [REST API] Fix incorrect `href` in `/:collection/:object/tasks`
+- [Plugin/Load balancer] Preventing the creation of a density plan when simple plan is selected in configuration. (PR [#7358](https://github.com/vatesfr/xen-orchestra/pull/7358))
```

Comment:
```suggestion
- [Plugin/load-balancer] Correctly create a *simple* instead of a *density* plan when it is selected (PR [#7358](https://github.com/vatesfr/xen-orchestra/pull/7358))
```